cypress/e2e/awx/resources/credentials.cy.ts

```diff
@@ -34,16 +34,17 @@ describe('Credentials', () => {
   });
 
   describe('Credentials: List View', () => {
+    let credential: Credential;
     beforeEach(() => {
       cy.createAWXCredential({
         kind: 'machine',
         organization: awxOrganization.id,
         credential_type: 1,
       }).then((cred) => {
         credential = cred;
+        cy.giveUserCredentialsAccess(credential.name, user.id, 'Use');
```

Comment:
This command will cause downstream failure.

Reply:
Removed.